rules-tests/TypeDeclaration/Rector/ClassMethod/AddVoidReturnTypeWhereNoReturnRector/Fixture/skip_throws_only_method.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddVoidReturnTypeWhereNoReturnRector\Fixture;
+
+final class NeverMethod
+{
+    protected function getValues()
```

Comment:
same with here, the `ReturnNeverTypeRector` will change `void` to `never` when feature available, see 

https://getrector.com/demo/dde48f5c-7497-489c-9392-3043e45cead5

Reply:
hm ok, I moved he pre PR-existing test from a "skip" into a regular test which expects `:void` beeing added.